packages/venia-ui/lib/components/Checkout/addressForm.js

```diff
@@ -32,7 +32,7 @@ const AddressForm = props => {
 
     const talonProps = useAddressForm({
         fields,
-        initialValues: props.initialValues,
+        initialValues: props.initialValues ? props.initialValues : {},
```

Comment:
```suggestion
        initialValues: props.initialValues || {}
```

Reply:
This is already set by the `defaultProps`, why is this necessary?